### PR TITLE
feat: add alias (@/~) support in ava tests

### PR DESCRIPTION
**Pull request by vinayakkulkarni**
- [x] Mentioned about this feature in PR #283 

Refer: https://github.com/nuxt/create-nuxt-app/pull/283#discussion_r305201318

Basically allows users to import Vue SFCs in tests using aliases rather than the `../../` (relative?) imports.

### CODE REVIEW

File: template/frameworks/ava/babel.config.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+module.exports = {
```
Comment (clarkdo) [inline template/frameworks/ava/babel.config.js]: Adding `babel.config.js` may have conflicts if other frameworks also needs it.

Can we put the config in `babel` of `ava.config.js` ? https://github.com/avajs/ava/blob/master/docs/recipes/babel.md#customize-how-ava-compiles-your-test-files

Comment (vinayakkulkarni) [reply template/frameworks/ava/babel.config.js]: - [x] Fixed in 4c26ca1

**Comment by clarkdo** [comment]
Conflict needs to be resolved

**Comment by vinayakkulkarni** [comment]
> Conflict needs to be resolved

Done :)

**Comment by vinayakkulkarni** [comment]
Thoughts on merging this?

**Pull request merged**